msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/ltw/TestCase2571361.java

```diff
@@ -0,0 +1,144 @@
+@LTWTests
+@RunOnAPI29Minus
+@SupportedBrokers(brokers = {BrokerLTW.class})
+public class TestCase2571361 extends AbstractMsalBrokerTest {
+
+    @Test
+    public void test_2571361() throws Throwable {
+        final String username = mLabAccount.getUsername();
+        final String password = mLabAccount.getPassword();
+
+        // AcquireToken interactively on OneAuthTestApp
+        final OneAuthTestApp oneAuthTestApp = new OneAuthTestApp();
+        oneAuthTestApp.uninstall();
+        oneAuthTestApp.install();
+        oneAuthTestApp.launch();
+        oneAuthTestApp.handleFirstRun();
+
+        final FirstPartyAppPromptHandlerParameters promptHandlerParameters = FirstPartyAppPromptHandlerParameters.builder()
+                .broker(mBroker)
+                .prompt(PromptParameter.LOGIN)
+                .loginHint(username)
+                .consentPageExpected(false)
+                .speedBumpExpected(false)
+                .sessionExpected(false)
+                .expectingBrokerAccountChooserActivity(false)
+                .expectingLoginPageAccountPicker(false)
+                .enrollPageExpected(false)
+                .build();
+        oneAuthTestApp.addFirstAccount(username, password, promptHandlerParameters);
+        oneAuthTestApp.confirmAccount(username);
+
+        // Install new Company Portal
+        final BrokerCompanyPortal brokerCompanyPortal = new BrokerCompanyPortal();
+        brokerCompanyPortal.uninstall();
+        brokerCompanyPortal.install();
+
+        // Install new MsalTestApp
+        final MsalTestApp msalTestApp = new MsalTestApp();
+        msalTestApp.uninstall();
+        msalTestApp.install();
+        msalTestApp.launch();
+        msalTestApp.handleFirstRun();
+
+        final MicrosoftStsPromptHandlerParameters promptHandlerParametersMsal = MicrosoftStsPromptHandlerParameters.builder()
+                .prompt(PromptParameter.SELECT_ACCOUNT)
+                .loginHint(username)
+                .sessionExpected(false)
+                .broker(mBroker)
+                .expectingBrokerAccountChooserActivity(false)
+                .expectingProvidedAccountInBroker(false)
+                .expectingLoginPageAccountPicker(false)
+                .expectingProvidedAccountInCookie(false)
+                .consentPageExpected(false)
+                .passwordPageExpected(false)
+                .speedBumpExpected(false)
+                .registerPageExpected(false)
+                .enrollPageExpected(false)
+                .staySignedInPageExpected(false)
+                .verifyYourIdentityPageExpected(false)
+                .howWouldYouLikeToSignInExpected(false)
+                .build();
+
+        // Add login hint and click on acquireToken button
+        // will not be prompted to enter credentials
+        msalTestApp.handleUserNameInput(username);
+        final String token = msalTestApp.acquireToken(username, password, promptHandlerParametersMsal, false);
```

Comment:
in this step, is the last param "false" determining whether you expect to see password prompt or not?

Reply:
yes, if it is true UI will handle password input, otherwise it is assumed that the password input interface will not appear